### ISSUE TITLE
[$10][MSFT-75] The NVDA doesn't read "Facebook/ Twitter/ LinkedIn/ Instagram " icons name when user try to mouse hover on them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2789]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges?
2. Scroll to the footer
3. Try to mouse hover on "Facebook/ Twitter/ LinkedIn/ Instagram " icons

**Expected Result**
The NVDA should read "Facebook/ Twitter/ LinkedIn/ Instagram " icons name when user try to mouse hover on them

**Actual Result**
The NVDA doesn't read "Facebook/ Twitter/ LinkedIn/ Instagram " icons name when user try to mouse hover on them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F10

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096065 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096065 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096065 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096065 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@codeMinter PR https://github.com/topcoder-platform/community-app/pull/2975

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097400 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097400 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
Verified on Chrome/Edge Browser.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097400<br/><br/>```This is an automated message for crazyk via Topcoder X```
